# Define the goal
float32 pos_z
---
# Define the result
bool trigger_off
---
# Define a feedback message
bool is_active